package.json

```diff
@@ -71,7 +71,7 @@
     "axios": "^0.17.1",
     "babel-polyfill": "^6.26.0",
     "bootstrap-sass": "^3.3.7",
-    "brace": "^0.10.0",
+    "brace": "^0.11.1",
```

Comment:
is it required? I think its actually now bundled in react-ace and we can remove it from here?